go/enclave/limiters/rolluplimiter.go

```diff
@@ -0,0 +1,45 @@
+package limiters
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/rlp"
+)
+
+var (
+	ErrFailedToEncode   = errors.New("failed to encode data")
+	ErrSizeExceedsLimit = errors.New("data size exceeds remaining limit")
+)
+
+// MaxTransactionSizeLimiter - configured to be close to what the ethereum clients
+// have configured as the maximum size a transaction can have. Note that this isn't
+// a protocol limit, but a miner imposed limit and it might be hard to find someone
+// to include a transaction if it goes above it
+// todo - figure out the best number, optimism uses 132KB
+const MaxTransactionSize = 64 * 1024
```

Comment:
This needs to be adjusted by a factor of 50% or something based on the compression rate.
(basically doubled)